### ISSUE TITLE
Can't Sustain "Scale and Fill client window" Preference

### ISSUE THREAD

**Issue by Lonniebiz** [FreeRDP/Remmina#557]
giox069 recently, and graciously, added a feature where you can scale a remote session to "[Fill client window when scaled](https://github.com/FreeRDP/Remmina/issues/537)". This feature allows you to scale a remote session without strictly respecting the native resolution's aspect ratio. This is my preferred way to see each remote session I launch.

The feature works great!

However, there is one problem. I cannot sustain this feature as my default preference and I have to manually select it:
1) Each time I launch Remmina
2) Each time I launch a new remote session from an already launched Remmina

The steps I have to repeat each time are:
1) Click the "Scale the desktop" button
2) Click the "Scaling options" button
3) Select the "Fill client when scaled" option (you've added)

This might not seem like much of a burden, and it isn't for people who launch one remote session in a day. However, if you launch 50 in a day, that's 150 additional steps.

I've always wished that Remmina would persist my preferences so that both 1) "Between sessions" and 2) "during sessions" it would always persist my preference of launching new remotes sessions as "scaled without respect to aspect ratio". Otherwise, I have to repeat these step for each remote desktop I launch (several times per day).

Even in the earlier versions, I've always had to click the "Scale" button for each session I launch. Now, I have to do that and also select the feature you've added (two additional clicks per remote session).

Look at the preferences dialog:
http://neartalk.com/ss/2015-04-10_007_813x531.png

I have Scrolled Window selected, but there is no option for:
"Full Size in Tab and Scaled Without Respect to Aspect Ratio"

In summary, there two preferences that cannot be sustained in Remmina-Next:
1) Can't sustain the preference of launching all remote sessions as "Scaled".
2) Can't sustain the preference of launching all remote sessions as ""Fill client window when scaled".

**Comment by giox069** [comment]
Are you using "Quick connect" or a "Saved profile" ? See the image here:
![remmina remote desktop client_001](https://cloud.githubusercontent.com/assets/5477803/7162080/de08fd28-e392-11e4-94dc-e7967cffe8df.png)

Connection in **saved profiles** store their settings every time you close a connection. Also please check to have the following option enabled in the remmina global preferences:

![remmina preferences_002](https://cloud.githubusercontent.com/assets/5477803/7162188/901b2342-e393-11e4-9833-8ac0f7458903.png)

**Comment by Lonniebiz** [comment]
Wow! That works now. I had disabled that option at one point due to [a bug](https://bugs.launchpad.net/ubuntu/+source/remmina/+bug/1370014) in a previous version.

I have over 200 saved profiles. Doing this only one time for each isn't so bad. Now, I'm fine with upgrading to Ubuntu 15.04. If not for Remmina-Next, I would have to stay on 14.04.

Ideally, when that option is NOT selected, you'd think there would be an option to set a default behavior for all profiles. But, what you've explained is certainly good enough for me.

Having an "Override Preferences" would be ideal, but I'm satisfied. THANK YOU!

**Comment by giox069** [comment]
I'm happy you find a way to make it work. We could also put a global preference to be used "by default" when using quick connect and when creating a new connection profile. But this will further make confusion to the user: too many options which do the same thing in different places are just confusing, as you could see. Adding a "override" as you propose, will give a third choiche on every visualizaiont option (1:global defaults, 2: saved, 3: ovverride) and this will make a total confusion to the user which never deeply analyzed/studied the "remmina preference system" before (98% of user).
So don't agree on introducing a new override option or other default options :smile:
